### PR TITLE
Improve install info in README

### LINKED ISSUE #238: README install instructions
Add install instructions, similar to omero-py.

### PR DESCRIPTION
**Pull request by will-moore**
Fixes #238

Improve install info in README.

I would *like* to also provide simple instructions for production deployment, but I'm not sure if that's possible?

cc @manics

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -26,12 +26,27 @@ Direct dependencies of OMERO.web are:
...
+    conda activate myenv
+    pip install omero-web
+
+Setting of the environment variable ``OMERODIR`` is required.
+``$OMERODIR/var/log/`` directory will contain log files.
+``$OMERODIR/etc/grid/config.xml`` is used to store config::
+
+    export OMERODIR=$(pwd)
+
+See: `OMERO`_ documentation for more details.
```
Comment (jburel) [inline README.rst:43]: this link does not give any details about the installation.
Either rephrase or remove the sentence

File: README.rst [hunk 2]
```diff
@@ -26,12 +26,27 @@ Direct dependencies of OMERO.web are:
...
 
 Usage
 -----
 
-See: `OMERO`_ documentation
+For running omero-web in production with nginx, see `OMERO`_ documentation.
```
Comment (jburel) [inline README.rst:48]: The link is not useful, it points to a general intro page

File: README.rst [hunk 3]
```diff
@@ -41,25 +56,39 @@ See: `OMERO`_ documentation
 Developer installation
 ----------------------
 
-OMERO.web depends on OMERO.py. If you want a developer installation of OMERO.py, replace ``pip install omero-py``
-with instructions at https://github.com/ome/omero-py.
-
-For a development installation we recommend creating a virtualenv with the following setup (example assumes ``python3.6`` but you can create and activate the virtualenv using any compatible Python):
+For a development installation we recommend creating a virtualenv as described above.
```
Comment (jburel) [inline README.rst:59]: virtual environment

**Comment by jburel** [comment]
For production installation, pointing to the installation doc as you did seems acceptable. I will not overload this README.

**Comment by jburel** [comment]
@will-moore Are you planning to push more changes?
It is already an improvement

**Comment by will-moore** [comment]
No. If it's good to merge then please go ahead, thanks.

**Pull request merged**